upup/pkg/kutil/delete_cluster.go

```diff
@@ -565,6 +565,10 @@ func DeleteSecurityGroup(cloud fi.Cloud, t *ResourceTracker) error {
 		}
 		response, err := c.EC2().DescribeSecurityGroups(request)
 		if err != nil {
+			if awsup.AWSErrorCode(err) == "InvalidGroup.NotFound" {
```

Comment:
The code shouldn't change; it would break a lot of clients.  I agree aws-sdk-go should export it, but they don't.  Arguably though, what would they call it - `InvalidGroupNotFound`?